html/pfappserver/lib/pfappserver/Form/Field/RE2.pm

```diff
@@ -0,0 +1,73 @@
+package pfappserver::Form::Field::RE2;
+
+=head1 NAME
+
+pfappserver::Form::Field::RE2 - A RE2 regex field
+
+=head1 DESCRIPTION
+
+This field extends the default Text field and checks if the input value is an valid RE2 regex
+
+=cut
+
+use HTML::FormHandler::Moose;
+extends 'HTML::FormHandler::Field::Text';
+
+use pf::util;
+use namespace::autoclean;
+
+
+our $class_messages = {
+    'regex' => 'It must be a valid regex',
```

Comment:
Perhaps saying "valid RE2 regex" to give a clue to the user what spec to look at for his regex